pkg/pb/metainfo.proto

```diff
@@ -387,7 +387,6 @@ message SegmentPosition {
 
 message SegmentBeginRequest {
     bytes           stream_id = 1 [(gogoproto.customtype) = "StreamID", (gogoproto.nullable) = false];
-    SegmentPosition position = 2;
```

Comment:
I think we need to keep segment position here to avoid later problems with tracking partial segments.

Reply:
The issue is that with the last segment we know its the last one after upload.

Reply:
Not after multipart upload.

Reply:
I see, I need figure out how to handle last segment when `position` is with `SegmentBeginRequest`

Reply:
@egonelbre at the moment I have only one idea to send each segment with normal indexes (from 0 to n) and with `CommitObject` rename the last one to `l/`.